lib/standalone.js

```diff
@@ -85,6 +85,17 @@ async function standalone({
 		return Promise.reject(error);
 	}
 
+	// convert e.g. c:\foo\bar to C:/foo/bar to avoid not working "extends", "plugins" and "customSyntax"
+	if (configBasedir) {
+		if (configBasedir[0] && configBasedir[1] === ':') {
```

Comment:
opinion

1. organisation: the path handler/fixer should probably be in a helper/file in `lib/utils/`.
2. implementation: we should probably use builtin fs methods—maybe `fs.realpathSync.native`?—or an existing npm package.
3. consistency: all known/similar cases need to be fixed at the same time

Reply:
[ask] Can we use [`normalize-path`](https://www.npmjs.com/package/normalize-path)? We've already used it.

Reply:
Yes, `normalize-path` could be used to fix part with backward slashes but when debugging thoroughly again problem is just lower-cased drive letter, so part with changing slashes I have just removed completely